lib/src/main/java/vmath/Vec3.java

```diff
@@ -29,12 +29,16 @@ public float z() {
     }
 
     public float mag() {
-        return (float) Math.sqrt(x*x + y*y + z*z);
+        return (float) Math.sqrt(x * x + y * y + z * z);
```

Comment:
I think this isn't part of the PR

Reply:
I agree with @vgdaut. I have also updated the Code Style document to mention that I want no whitespace around multiplication and division, as this was a decision made in a previous lab.